test/actors/SessionsSchedulerSpec.scala

```diff
@@ -69,6 +69,8 @@ class SessionsSchedulerSpec(_system: ActorSystem) extends TestKit(_system: Actor
         cancelled = false,
```

Comment:
Would you want to change youtubeLink and slideShareLink to youtubeURL and slideShareURL in test cases as well?